consultation_analyser/support_console/ingest.py

```diff
@@ -223,3 +332,57 @@ def import_all_responses_from_jsonl(
     # Any remaining lines < batch size
     if lines:
         import_responses_job.delay(question_part=question_part, responses_data=lines)
+
+
+def import_themes_from_json(
+    question_part: QuestionPart, bucket_name: str, question_part_folder_key: str
+) -> None:
+    s3 = boto3.client("s3")
+    data_key = f"{question_part_folder_key}themes.json"
+    response = s3.get_object(Bucket=bucket_name, Key=data_key)
+    theme_data = json.loads(response["Body"].read().decode("utf-8"))
+    import_themes(question_part=question_part, theme_data=theme_data)
+
+
+def import_all_theme_mappings_from_jsonl(
+    question_part: QuestionPart, bucket_name: str, question_part_folder_key: str, batch_size: int
+) -> None:
+    logger.info(
+        f"Importing theme_mappings from {question_part_folder_key}, batch_size {batch_size}"
+    )
+    theme_mappings_file_key = f"{question_part_folder_key}mapping.jsonl"
+    s3_client = boto3.client("s3")
+    response = s3_client.get_object(Bucket=bucket_name, Key=theme_mappings_file_key)
+    lines = []
+    for line in response["Body"].iter_lines():
+        lines.append(line)
+        if len(lines) == batch_size:
```

Comment:
Aware that this is my code! 

Not for this PR, but I feel like there may be a cleaner/more Pythonic way of doing this, so do go ahead and change if you can think of one!

Reply:
great, have set this up as Trello ticket 346